libs/sdk-ui-tests-e2e/docker-compose-isolated.yaml

```diff
@@ -43,7 +43,7 @@ services:
             - backend-mock
 
     backend-mock:
-        image: wiremock/wiremock:2.35.0
```

Comment:
Please keep the same version. If you want to bump-up, ensure

- All repo are the same
- New version should work well by run all available test (isolated + recording)

Reply:
Login into OKta -> AWS -> INFRA
Go to link https://us-east-1.console.aws.amazon.com/ecr/repositories/private/020413372491/3rdparty/wiremock/wiremock?region=us-east-1 to see all available tag